This is to stop pip check complaining.

mkl isn't a python package for us so there's no metadata to support pip check.
Index: pytorch/setup.py
===================================================================
--- pytorch.orig/setup.py	2024-05-03 14:37:02.824162543 -0500
+++ pytorch/setup.py	2024-05-29 11:37:14.428877813 -0500
@@ -1111,7 +1111,6 @@
         "networkx",
         "jinja2",
         "fsspec",
-        'mkl>=2021.1.1,<=2021.4.0; platform_system == "Windows"',
     ]
 
     # Parse the command line and check the arguments before we proceed with
